crates/burn-tch/README.md

```diff
@@ -243,15 +243,32 @@ is to use a PyTorch installation. This requires a Python installation.
 _Note: MPS acceleration is available on MacOS 12.3+._
 
 ```shell
-pip install torch==2.2.0
+pip install torch==2.2.0 numpy==1.26.4 setuptools
 export LIBTORCH_USE_PYTORCH=1
 export DYLD_LIBRARY_PATH=/path/to/pytorch/lib:$DYLD_LIBRARY_PATH
 ```
 
+**Note:** if `venv` is used, it should be activated during coding and building,
+or the compiler may not work properly.
+
 ## Example Usage
 
 For a simple example, check out any of the test programs in [`src/bin/`](./src/bin/). Each program
 sets the device to use and performs a simple element-wise addition.
 
 For a more complete example using the `tch` backend, take a loot at the
 [Burn mnist example](https://github.com/tracel-ai/burn/tree/main/examples/mnist).
+
+## Too many environment variables?
+
+Try `.cargo/config.toml` ([cargo book](https://doc.rust-lang.org/cargo/reference/config.html#env)):
+
+For example, Linux.
+```bash
+mkdir .cargo
+cat <<EOF > .cargo/config.toml
+[env]
+LD_LIBRARY_PATH = "$(pwd)/libtorch/lib:$LD_LIBRARY_PATH"
+LIBTORCH = "$(pwd)/libtorch"
+EOF
+```
```

Comment:
Instead of giving commands to write this to the file, I think I would put a simple sentence to explain the steps (and use absolute path instead of `$(pwd)` which may not always be the case for everyone).

Something like this maybe?

## Too many environment variables?

Instead of setting the environments in your shell, you can add them to your `.cargo/config.toml`

```
[env]
LIBTORCH = /absolute/path/to/libtorch/
LD_LIBRARY_PATH = /absolute/path/to/libtorch/lib:$LD_LIBRARY_PATH
```

Reply:
> LD_LIBRARY_PATH = /absolute/path/to/libtorch/lib:$LD_LIBRARY_PATH

In `config.toml`, `..libtorch$LD_LIBRARY_PATH` may not work and the value should be wrapped with `"`. I'll test and modify.

Reply:
I tested, and that's true the `$LD_LIBRARY_PATH` will be keep as `$LD_LIBRARY_PATH` in env vars. 
```
[tagger/src/lib.rs:19:9] ld = "/run/media/louis/ZhiTai/rust/burnxp/libtorch:$PATH"
```
---

```toml
[env]
LD_LIBRARY_PATH = "/absolute/path/to/libtorch/lib"
LIBTORCH = "/absolute/path/to/libtorch/libtorch"
```

Maybe this is Okey? As on my machine, `LD_LIBRARY_PATH` is actually empty. 

Or recommending a bash script like `cat<<EOF > file ... EOF` is also okey. I see `kubernetes`' documentation uses such method too.

Reply:
Ahhh yeah you're right.. I definitely didn't think this one through before 😅 

In that case, a combination of both might work best. Leave the instructions with the dummy example `/absolute/path/to/libtorch` and under we can give the instructions to prepend it to the variable while keeping your existing var!